packages/fresh/src/test_utils.ts

```diff
@@ -137,10 +137,15 @@ export async function withTmpDir(
   return {
     dir,
     async [Symbol.asyncDispose]() {
+      // Skip pointless cleanup in CI, speed up tests
+      if (Deno.env.get("CI") === "true") return;
```

Comment:
GitHub Actions (and most other CI tools) sets this to `true` when run.

https://docs.github.com/en/actions/reference/workflows-and-actions/variables#default-environment-variables